cmd/tmsp-cli/tmsp-cli.go

```diff
@@ -146,19 +207,22 @@ func cmdBatch(app *cli.App, c *cli.Context) error {
 		} else if err != nil {
 			return err
 		}
-		args := []string{"tmsp-cli"}
-		if c.GlobalBool("verbose") {
-			args = append(args, "--verbose")
+
+		args := persistentArgs(line)
+		err2 := app.Run(args) //cli prints error within its func call
```

Comment:
I prefer to distinguish the name to make the code less ambiguous. It can also help simplify debugging for future code. I'd opt to maintain use of `err2`